changelog/unreleased/10193

```diff
@@ -0,0 +1,5 @@
+Change: CMD Client, don't guess remote folder
+
+The commandline client was modified to explicitly accept remote folder, the must no longer be encoded in the server url.
```

Comment:
I'm not sure what you're trying to say...?

Reply:
missing subject